layouts/_default/list.html

```diff
@@ -53,23 +50,15 @@ <h2 class="post-title"><a href="{{ .RelPermalink }}">{{ .Title }}</a></h2>
 						  {{end}}
 						  {{end}}
 						 </span>
-						{{ .Summary | safeHTML }}
-						 <footer class="read-more">
-						<a href="{{ .RelPermalink }}">Read more&raquo;</a>
-							   
-
-
-
-						 </footer>
-
-				
+						{{ .Params.Subtitle | safeHTML }}
```

Comment:
why change `.Summary` to `.Params.Subtitle`? what it does?

Reply:
So this is the part that is shown when viewing a list of posts. If the post includes:

```
content
<!--more-->
more content
```

Then the section that appears before the `more` tag is taken as the post summary, and is used for the list. If that tag isn’t specified, then Hugo tries to “guess” as to what the summary should be.

This change would make the post summary the “subtitle” instead.

I suppose it’s up for debate. Do we make the post summary the article subtitle, or do we add the `more` tag to the post template so people know to use it?

Also, to see the difference: https://jellyfin.org/posts/

My post uses the explicit summary break, Josh’s didn’t.

Reply:
I forgot to remove this actually. My goal was to improve the summaries a bit because they seemed a bit short in the list view, but this didn't fix much.

Reply:
The better fix then is to add the summary tag to the post template, and also add it in to Josh’s post. Changing it in mine is okay too, but it was a short post to begin with.

Reply:
@dkanada Should we move this back then? I can make a PR to fix up the summary tag in the template and Josh's post too.